library/src/main/java/com/novoda/downloadmanager/NotificationManager.java

```diff
@@ -0,0 +1,22 @@
+package com.novoda.downloadmanager;
+
+import android.app.Notification;
+import android.support.annotation.Nullable;
+import android.support.v4.app.NotificationManagerCompat;
+
+class NotificationManager {
```

Comment:
Wrapping so that we can mock and test the `ServiceNotificationDispatcher`

Reply:
can't we mock `notificationManagerCompat` in tests? This wrapping seems to be unnecessary.

Reply:
No, it's final unfortunately. I did take a look 😢

Reply:
`Mockito` is able to mock final classes since version 2.x
https://github.com/mockito/mockito/wiki/What%27s-new-in-Mockito-2#unmockable

Reply:
Ah good point! Forgot they added that incubating feature. d352695